README.md

```diff
@@ -60,4 +80,10 @@ Regardless of logged-in state or permissions, the Craft Debug Bar will always be
 
 ### Front-end edit entry Links
 
-When visiting an element URL on the front-end, an edit entry link to edit that element in the CP will be present in the lower left hand corner. This will always be present in the dev environment regardless of logged-in state or permissions.
\ No newline at end of file
+When visiting an element URL on the front-end, an edit entry link to edit that element in the CP will be present in the lower left hand corner. This will always be present in the dev environment regardless of logged-in state or permissions.
+
+### Components exposed to Twig `craft.viget` sub-object
+
+#### `craft.viget.util.fillInEntries()`
+
+This method queries for additional, deduped entries to a fill a number of needed entries
```

Comment:
How about documenting the parameters, and mentioning that if more than `limit` `entries` are passed in the result will be more than `limit` (a good flexible behavior but worth calling out)

Reply:
Need to figure out documentation overall, this is just carried over from the starter module. Likely will go a docs folder route